docs/download.rst

```diff
@@ -173,15 +174,20 @@ Downloading API
 
    :query _refresh: use ``_refresh=1`` to force Tecken to update cache
 
+   :resheader Location: redirect location for the file
+   :resheader Content-Length: length in bytes for the file
+   :resheader Content-Encoding: (optional) content encoding for the file; note
+       that ``.sym`` files are compressed even thought he file extension doesn't
+       indicate that
+
    :statuscode 302: symbol file was found--follow redirect url in ``Location`` header in
        the response to get to the final url
-   :statuscode 400: requested symbol file has bad characters
+   :statuscode 400: param values have bad characters in them or are otherwise invalid
    :statuscode 404: symbol file was not found
-   :statuscode 429: sleep for a bit and retry
-   :statuscode 500: sleep for a bit and retry; if retrying doesn't work, then please
-       file a bug report
-   :statuscode 503: sleep for a bit and retry; if retrying doesn't work, then please
-       file a bug report
+   :statuscode 429: your request has been rate-limited; sleep for a bit and retry
+   :statuscode 500: there's an error with the server; sleep for a bit and
+       retry; if retrying doesn't work, then please file a bug report
+   :statuscode 502: sleep for a bit and retry
```

Comment:
The logs are from nginx, so they only include requests that make it to nginx. If the ELB can't reach the backend, it will reply with a 503 or 504, and these requests won't be in the logs. It's possible to enable ELB logs as well, but we never did that for a variety of reasons. In GCP, we'll also have load balancer logs, so we'll be able to see all the requests.